application/controllers/artifacts_controller.py

```diff
@@ -10,21 +10,22 @@
 from ..responders import no_content, respond_with
 from ..errors import NotFound
 from ..error_handling.es_connection import check_es_connection
+from ..models.elastic_artifact import ElasticArtifact
 from ..models.artifact import Artifact
 from ..validators import artifacts_validator
 from ..recognition.image_recognition import ImageRecognizer
 from .application_controller import ApplicationController
 
+
 class ArtifactsController(ApplicationController):
     """ Controller for Artifacts """
 
     method_decorators = [check_es_connection]
 
     def show(self, object_id):
         """Logic for getting a single artifact"""
-
         try:
-            artifact = Artifact.find(object_id)
+            artifact = ElasticArtifact.find(object_id)
```

Comment:
Why do you use ElasticArtifact here? I though the idea was to 
always call the wrapper. Even if it just calls ElasticArtifact, makes it easyer to replace.